pipelines/tools-staging-prod-infra.yaml

```diff
@@ -0,0 +1,422 @@
+
+groups:
+- name: provision
+  jobs:
+  - terraform
+  - bootstrap-tools-cluster
+  - apply-addons-tools-cluster
+  - bootstrap-staging-cluster
+  - apply-addons-staging-cluster
+  - bootstrap-prod-cluster
+  - apply-addons-prod-cluster
+- name: destroy
+  jobs:
+  - destroy-tools-cluster
+  - destroy-staging-cluster
+  - destroy-prod-cluster
+
+
+
+
+
+task_image_resource: &task_image_resource
+  type: docker-image
+  source: {repository: "govsvc/task-toolbox", tag: "1.0.0"}
+
+
+apply_addons_task: &apply_addons_task
+  platform: linux
+  image_resource: *task_image_resource
+  params: &apply_addons_task_params
+    ACCOUNT_ROLE_ARN: ((account-role-arn))
+    ACCOUNT_NAME: ((account-name))
+    CLUSTER_NAME:
+    TERRAFORM_BUCKET: cd-gsp-private-qndvvc
```

Comment:
I feel like we should be prefixing our s3 bucket names with `gds` or something similar. Maybe something to raise with gds-way?

Reply:
This bucket is part of the deployer concourse handed down to us by the autom8 elders